### PR TITLE
fix: hasPacketReciept method removed

### PR DESCRIPTION
**Pull request by gcranju**
### Commit Message

```bash
type: commit message
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by AntonAndell** [comment]
@izyak @viveksharmapoudel  Which one does the relay use?

**Review by izyak — APPROVED** [review]

**Comment by izyak** [comment]
It used getPacketReceipt method

**Review by redlarva — CHANGES_REQUESTED** [review]
We should update the `getPacketReceipt` method. Currently, if the `portId` or `channelId` doesn't exist, it throws an error. I suggest changing this behavior to have it return false when `portId` or `channelId`  does not exists. This issue also extends to the CosmWasm ibc contract, where currently an error occurs if a packet is not found.

**Comment by redlarva** [comment]
Additionally ,in our e2e/integration tests, we currently use the hasPacketReceipt method to confirm if packets reached the target chain for both Icon and CosmWasm contracts. Instead of hasPacketReceipt, let's use the getPacketReceipt method for validation in e2e/integration setup.

File: contracts/javascore/ibc/src/main/java/ibc/ics24/host/IBCStore.java [hunk 1]
```diff
@@ -123,7 +123,11 @@ public BigInteger getNextSequenceAcknowledgement(String portId, String channelId
 
     @External(readonly = true)
     public boolean getPacketReceipt(String portId, String channelId, BigInteger sequence) {
-        return packetReceipts.at(portId).at(channelId).get(sequence);
+        try {
+            return packetReceipts.at(portId).at(channelId).get(sequence);
+        } catch (Exception e) {
+            return false;
+        }
```
Comment (redlarva) [inline contracts/javascore/ibc/src/main/java/ibc/ics24/host/IBCStore.java:130]:
```suggestion
           return packetReceipts.at(portId).at(channelId).getOrDefault(sequence, false);
```
should works fine

**Review by redlarva — APPROVED** [review]
looks good

**Pull request merged**